src/include/libpq/libpq-be.h

```diff
@@ -250,6 +250,7 @@ typedef struct Port
 	SSL		   *ssl;
 	X509	   *peer;
 #endif
+	bool		is_tds_conn;
```

Comment:
this information is available on TSQL side, we should add hook rather than adding a new member here.

Reply:
We rather keep it as it is for some other reasons.